composer.json

```diff
@@ -77,6 +77,7 @@
         "mockery/mockery": "^1.0",
         "nunomaduro/collision": "^7.0",
         "orchestra/testbench": "^7.8 || ^8.0 || ^9.0",
+        "phpstan/phpstan": "^1.12",
```

Comment:
Can/should we install and configure PHPStan in the `data-hub-foundation` package, so we can use it in all of our projects?

Reply:
Yes! I moved PHPStan into the data-hub-foundation and created a composer script to invoke it.